crates/aptos-protos/src/pb/aptos.transaction.v1.rs

```diff
@@ -373,6 +373,7 @@ pub mod transaction_payload {
         EntryFunctionPayload = 0,
         ScriptPayload = 1,
         ModuleBundlePayload = 2,
+        MultisigPayload = 3,
```

Comment:
did you update these manually? if so we need to revert this and do it properly via .proto change and buf generate.